FirebaseStorageUI/UIImageView+FirebaseStorage.h

```diff
@@ -0,0 +1,127 @@
+//
+//  Copyright (c) 2016 Google Inc.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//  http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+//
+
+@import UIKit;
+
+@import FirebaseStorage;
+@import SDWebImage;
+
+NS_ASSUME_NONNULL_BEGIN
+
+@interface UIImageView (FirebaseStorage)
+
+/**
+ * Returns the maximum image download size, in bytes. Defaults to 10e6.
+ */
++ (UInt64)sd_defaultMaxImageSize;
+
+/**
+ * Sets the maximum image download size, in bytes.
+ * @param size The new maximum image download size.
+ */
++ (void)sd_setDefaultMaxImageSize:(UInt64)size;
+
+/**
+ * The current download task, if the image view is downloading an image.
+ * Must be invoked on the main queue.
```

Comment:
It doesn't throw any errors, though, it's just undefined behavior. Like how [the UIView methods don't throw](https://developer.apple.com/reference/uikit/uiview).